### ISSUE TITLE
Yak is missing as wildshape option

### ISSUE THREAD

**Issue by Tywele** [safety-orange/Imports-for-MPMB-s-Character-Sheet#81]
The yak is missing as a wildshape option in [pub_20160906_SKT.js](https://github.com/safety-orange/Imports-for-MPMB-s-Character-Sheet/blob/master/WotC%20material/pub_20160906_SKT.js): https://5e.tools/bestiary.html#yak_skt

**Comment by Tywele** [comment]
In the meantime I created the list entry for myself:

```Javascript
CreatureList["yak"] = {
    name: "Yak",
    source: [["SKT", 172]],
    size: 2,
    type: "Beast",
    alignment: "unaligned",
    ac: 10,
    hp: 13,
    hd: [2, 10],
    speed: "50 ft",
    scores: [16, 10, 12, 2, 10, 6],
    senses: "",
    passivePerception: 10,
    challengeRating: "1/4",
    proficiencyBonus: 2,
    attacksAction: 1,
    attacks: [{
        name: "Ram",
        ability: 1,
        damage: [1, 6, "bludgeoning"],
        range: "Melee (5 ft)",
        description: ""
    }, {
        name: "Hooves",
        ability: 1,
        damage: [2, 4, "bludgeoning"],
        range: "Melee (5 ft)",
        description: ""
    }],
    traits: [{
        name: "Charge",
        description: "If the yak moves at least 20 feet straight toward a target and then hits it with a ram attack on the same turn, the target takes an extra 7 (2d6) damage. If the target is a creature, it must succeed on a DC 13 Strength saving throw or be knocked prone."
    }]
}
```

**Comment by safety-orange** [comment]
Thank you for reporting this issue. As the Yak is just the Elk by a different name, I'll get MPMB to include it in the built-in SRD code. Until that time, you can select "Elk" on a companion page and, if you are so inclined, change the race name to "Yak". This should keep the Elk stats but show a different name.

Alternatively, you can add this custom add-on script:

```js
CreatureList["elk"].nameAlt.push("Yak");
```

**Comment by safety-orange** [comment]
This has now been fixed in [v13.2.3 of MPMB's main GitHub repo](https://github.com/morepurplemorebetter/MPMBs-Character-Record-Sheet/releases/tag/v13.2.3).
